### PR TITLE
Adding spanish translation to homepage

### PR DESCRIPTION
**Pull request by FernandoRoldan93**
Readme file and `ARManual/.vuepress/config.js` translated to Spanish as asked in the issue #13

### CODE REVIEW

File: ARManual/es/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+home: true
+heroImage: /logo.svg
+actionText: Read →
```
Comment (domnantas) [inline ARManual/es/README.md:4]: `Read` needs translation too

**Review by domnantas — CHANGES_REQUESTED** [review]
Thank you! Only one translation is missing

File: ARManual/es/README.md (hunk 1 above)
Comment (FernandoRoldan93) [reply ARManual/es/README.md]: Ups, sorry! I have just add it 😅

**Review by domnantas — APPROVED** [review]
Gracias!

**Comment by FernandoRoldan93** [comment]
> Gracias!

Prašom! :slightly_smiling_face:

**Pull request merged**